README.md

```diff
@@ -129,10 +121,34 @@ curl -1sLf \
 sudo yum install redpanda
 ```
 
-## RC releases on Docker
+### RC releases on Docker
 
-This is an example with the `v23.1.1-rc1` version prior to the 23.1.1 release.
+Example with the `v23.1.1-rc1`. A version prior to the 23.1.1 release.
```

Comment:
```suggestion
Example with `v23.1.1-rc1`:
```

Reply:
should we use something more recent?

Reply:
Eh, I thought about this, but this will always be a cat-mouse scenario without automation. I don't mind leaving an old version there.